docs/install/openshift.md

```diff
@@ -28,89 +28,54 @@ Once the cluster is up and running, log in as the `cluster-admin` user.
 oc login -u kubeadmin
 ```
 
-The following section outlines the steps necessary to deploy the ArgoCD Operator manually using standard Kubernetes manifests.
+!!! info
+    Make sure you download the source code from release section: https://github.com/argoproj-labs/argocd-operator/releases. Compiling from the source code cloned off main repo may not provide the most stable result.
 
 ### Namespace
 
-It is a good idea to create a new namespace for the operator.
+By default, the operator is installed into the `argocd-operator-system` namespace. To modify this, update the
+value of the `namespace` specified in the `config/default/kustomization.yaml` file. 
```

Comment:
can we say replace namespace: argocd-operator-system with your namespace?

Reply:
I think this looks easy to understand :)